addons/sale_loyalty_delivery/models/sale_order.py

```diff
@@ -19,6 +19,14 @@ def _get_lines_impacting_invoice_status(self):
             lambda line: not line.is_reward_line
         )
 
+    def _get_not_rewarded_order_lines(self):
+        """Exclude delivery lines from consideration for reward points."""
+        order_line = super()._get_not_rewarded_order_lines()
+        if 'is_delivery' in order_line._fields:
+            return order_line.filtered(lambda line: not line.is_delivery)
+        else:
+            return order_line
```

Comment:
```suggestion
        return order_line.filtered(lambda line: not line.is_delivery)
```

Reply:
what if line won't have `is_delivery` attribute?

Reply:
If delivery is installed, it has, and it defaults to `False`.
`sale_loyalty_delivery` depends on delivery, so if this code is read, then the field exist =]